From d6e78fe580f612754503e0b4ffd075443cafa6fc Mon Sep 17 00:00:00 2001
From: Zygmunt Krynicki <me@zygoon.pl>
Date: Thu, 17 Jan 2019 15:51:14 +0200
Subject: [PATCH 6/9] systemd: disable snapfuse system

Upstream snapd uses an elaborate hack to bundle squashfuse under the
name snapfuse, and built as a fake go package. This component is not
available in Debian where bundling elements is not allowed.

Signed-off-by: Zygmunt Krynicki <me@zygoon.pl>
---
 systemd/systemd.go | 2 --
 1 file changed, 2 deletions(-)

diff --git a/systemd/systemd.go b/systemd/systemd.go
index f1863732a..d4e09a9a0 100644
--- a/systemd/systemd.go
+++ b/systemd/systemd.go
@@ -33,8 +33,6 @@ import (
 	"sync/atomic"
 	"time"
 
-	_ "github.com/snapcore/squashfuse"
-
 	"github.com/snapcore/snapd/dirs"
 	"github.com/snapcore/snapd/osutil"
 	"github.com/snapcore/snapd/osutil/squashfs"
-- 
2.17.1

